.github/workflows/continuous-deployment.yml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Deployment
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  build:
+    name: Build the docker image
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to docker hub
+        uses: docker/login-action@v1
+        with:
+          username: tomansion
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build the Docker image
+        run: docker build . -t museumgo_app
+
+      - name: Tag the Docker image with latest and the tag name
+        run: |
+          echo ${{ github.sha }}
+          docker tag museumgo_app tomansion/museumgo_app:latest
+          docker tag museumgo_app tomansion/museumgo_app:${{ github.sha }}
+
+      - name: Upload Docker image to Docker Hub
+        run: docker push tomansion/museumgo_app --all-tags
+
+  deploy:
+    needs: build
+    name: Deploy to hosted server
+    runs-on: ubuntu-latest
```

Comment:
Should we define a specific version instead ?

Reply:
Fixed in all jobs in [9f04eeb](https://github.com/Museum-GO/MuseumGO-App/pull/32/commits/9f04eeb45c43c967dcace29f43f5e2acfd177999)